Sources/Model/TransferMethod/HyperwalletBankAccountQueryParam.swift

```diff
@@ -48,10 +34,14 @@ public class HyperwalletBankAccountQueryParam: HyperwalletTransferMethodQueryPar
 
     override public func toQuery() -> [String: String] {
         var query = super.toQuery()
+
         if let type = type {
-            query[QueryParam.type.rawValue] = type.rawValue
+            if type.rawValue == HyperwalletTransferMethod.TransferMethodType.bankAccount.rawValue {
+                  query[QueryParam.type.rawValue] = HyperwalletTransferMethod.TransferMethodType.bankAccount.rawValue
+            } else {
+                 query[QueryParam.type.rawValue] = HyperwalletTransferMethod.TransferMethodType.wireAccount.rawValue
+            }
```

Comment:
What if integrator selects prepaidCard type for HyperwalletBankAccountQueryParam, then it will set prepaidCard instead and that is wrong